basket/src/components/Profile.tsx

```diff
@@ -0,0 +1,28 @@
+type Session = {
+  loginUser: {
+    id: number;
+    name: string;
+  };
+  cart: Array<{ id: number; name: string; price: number }>;
+};
+
+type MyProps = {
+  session: Session;
+  logout: () => void;
+};
+
+export const Profile = ({ session, logout }: MyProps) => {
+  return (
```

Comment:
여기서도 한번 디스트럭처링 해서 사용해주면 객체에 접근할 때 뎁스를 줄일 수 있어서 코드를 훨씬 깔끔하게 작성할 수 있음!

```suggestion
export const Profile = ({ session, logout }: MyProps) => {
  const { loginUser, cart } = session;
  
  return (
```